rackhd_BIST/test_utils.py

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/env python
+
+# Copyright 2017, Dell EMC, Inc.
+# -*- coding: UTF-8 -*-
+
+"""
+    Test utilities for RackHD built in self test script.
+    This file packs frequently used functions and classes for RackHD BIST
+"""
+
+import os
+import sys
+import re
+import json
+import time
+import subprocess
+import logging
+
+LOGGER_NAME = "rackhd_bist_log"
+
+def get_configurations():
+    """
+    Get BIST configurations from configure files
+    """
+    rackhd_bist_config = robust_load_json_file("./bist_rackhd_config.json")
+    if rackhd_bist_config["exit_code"]:
```

Comment:
The `exist_code` may be 0, that means success. If you rename it to `error_code` that will be more clear.